students/saleta_sebastian/lesson_12_debugging/regex_search.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+
+logging.basicConfig(level=logging.DEBUG, format=(' %(asctime)s - %(levelname)s'
+                                                 ' - %(message)s'))
+logger = logging.getLogger('__name__')
+
+
+def set_logger_level(lvl):
```

Comment:
What should happen when user will no pass -v flag? Should info logs print?

Reply:
Now its printing. I think that if user not pass -v logs should be off. I will change this.